### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by justinegeffen**
As per discussion with @itao this PR renames Incident Response > Incident Management.

Note: 
- The doc URL is generated from the repo name. I'm not sure whether it's possible to change the repo name or have the Gitbook URL changed independently of the repo name. @lieut-data. 
- There are missing event descriptions in the telemetry page. @agarciamontoro, could you confirm whether we should remove for now?
- Events and fields in telemetry: These need to be confirmed and possibly updated (@itao) 

These fields need to be included in the telemetry docs but have been removed for now (@agarciamontoro):
| Restart incident |||
| Set assignee |||
| Update playbook|||
| Change commander |||
| Change stage|||
| Run checklist item slash command |||

### CODE REVIEW

File: docs/telemetry.md [hunk 1]
```diff
@@ -18,19 +18,13 @@ Every event we track is accompanied with metadata that help us identify each eve
...
... identifier of the team where the incident channel is created.</br><br>`CreatedAt`: Timestamp of the incident creation.</br><br>`ChannelIDs`: A list containing a single element, the channel created along with the incident.</br><br>`PostID`: Unique identifier of the post.</br><br>`NumChecklists`: Number of checklists. It always equals 1.</br><br>`TotalChecklistItems`: Number of checklist items this incident starts with. It always equals 0.</br><br>`ID`: Unique identifier of the incident.</br><br>`IsActive`: Boolean value indicating if the incident is active. It always equals `true`.</br><br>`CommanderUserID`: Unique identifier of the commander of the incident. It equals the identifier of the user that created the incident.</br><br>`TeamID`: Unique identifier of the team where the incident channel is created.</br><br>`CreatedAt`: Timestamp of the incident creation.</br><br>`ChannelIDs`: A list containing a single element, the channel created along with the incident.</br><br>`PostID`: Unique identifier of the post.</br><br>`NumChecklists`: Number of checklists. It always equals 1.</br><br>`TotalChecklistItems`: Number of checklist items this incident starts with. It always equals 0. |
```
Comment (justinegeffen) [inline docs/telemetry.md:21]: Can we rename "checklists" to "tasks"?

**Comment by agarciamontoro** [comment]
Sorry for not updating the telemetry events for now. If that can wait until Friday, I'll be able to do it then. Also, this is being reworked right now (https://mattermost.atlassian.net/browse/MM-29720 and https://mattermost.atlassian.net/browse/MM-29754, at least), so maybe it's a good idea to wait for that. What do you think?

File: docs/telemetry.md (hunk 1 above)
Comment (agarciamontoro) [reply docs/telemetry.md:21]: Looks like @itao suggested to rename checklists to stages and items to tasks, to be consistent. I'm ok with that.

**Comment by justinegeffen** [comment]
No problem at all, @agarciamontoro! I'm happy to wait for the changes and fixes but this PR should be merged prior to that as there are some formatting issues that are a priority to fix. 

We can open a new telemetry PR once this is merged if that works for you?

**Comment by agarciamontoro** [comment]
Absolutely! About the change of name in the repo, I think that's doable. Can you queue that item for tomorrow's meeting? I'm approving this now.

**Review by agarciamontoro — APPROVED** [review]
Management will be! :tada:

**Pull request merged**